BotModules/Pager/TwilioConfig.json

```diff
@@ -0,0 +1,6 @@
+{
+    "machineDetection": "Enable",
+     "twiml": "<Response><Say>Hello! I am active</Say></Response>",
+     "to": "+16234146654",
+     "from": "+12029723644"
```

Comment:
Are these numbers we're comfortable being public?

Reply:
Yea I think we should move this all to the .env file. It is more an environment or deployment configuration rather than a general coding configuration anyways.